iChef-WebUi/src/app/components/recipe-page/recipe-page.component.html

```diff
@@ -0,0 +1,128 @@
+<body>
+  <h1>Recipe Creation</h1>
+  <div class="left">
+   
+      <div class="form-field">
+        <input class="input" text-align="center" placeholder="Recipe Title">
+      </div>
+
+
+   <br>      
+          &nbsp;
+        <label class="form-field-smaller">
+          Ingredients: </label>&nbsp;&nbsp;
+        
+        <span class="form-field-smaller">
+          <input class="input-smaller" >
+        </span>&nbsp;&nbsp;
+        
+        <span class="form-field-smaller">
+          <input class="input-smaller">
+        </span>&nbsp;&nbsp;
+
+        <!-- <br> -->
+        <select name="Quantity" id="quantity">
+          <option value="g">g</option>
+          <option value="kg">kg</option>
+          <option value="ml">ml</option>
+          <option value="l">l</option>
+        </select>
+
+        <button class="plus">
+
+          +
+      
+        </button>
+
+        
+
+        <br><br>
+       
+          &nbsp;
+          <label class="form-field-smaller">
+            Utensils: </label>&nbsp;&nbsp;
+
+
+            <span class="form-field-smaller">
+              <input class="input-smaller">
+            </span>&nbsp;&nbsp;
+        
+            <button class="plus">
+
+              +
+          
+            </button>
+
+        <br><br>
+        
+          &nbsp;
+          <label class="form-field-smaller">
+            Time to prepare: </label>&nbsp;&nbsp;
+
+
+            <span class="form-field-smaller">
+              <input class="input-smaller">
+            </span>&nbsp;&nbsp;
+
+            <label class="form-field-smaller">
+              Difficulty: </label>&nbsp;&nbsp;
+  
+  
+              <select name="Difficulty" id="difficulty">
+                <option value="easy">easy</option>
+                <option value="medium">medium</option>
+                <option value="hard">hard</option>
+                
+              </select>&nbsp;&nbsp;
+      
+        <br><br>
+       <div class="field">
+
+        <p>Preparation Instructions:</p>
+
+        <textarea rows = "8" cols = "85" name = "instructions">
+
+        </textarea>
+
+      </div>
+     <br><br>
+     <div class="field">
+      <p> Extra Notes: </p>
+      <textarea rows = "8" cols = "85" name = "extranotes">
+     </textarea>
+
+     </div>
+
+  </div>
+
+
+  
+  
+  <div class="right">
+    
+      <div class="up">
+          <h2>Ingredients/Utensils Overview</h2>
+          <div class = "ingredients-overview-left">
+            <p>Ingredients</p>
+
+          </div>
+
+          <div class= "utensils-overview-right">
+            <p>Utensils</p>
+          </div>
+
+
+      </div>
+      <div class="down">
+
+      </div>
+   
+  </div>
+
+  <button class="save">
```

Comment:
Would be nice if the class was named 'save-btn' to explicitly specify that this class is used for styling a button